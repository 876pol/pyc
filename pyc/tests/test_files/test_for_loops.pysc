int main() {
    for (int i = 0; i < 5; i += 1) print((string) i);
    print("\n");

    for (int i = 0; i < 10; i += 2) {
        if (i == 6) break;
        print((string) i);
    }
    print("\n");

    for (int i = 0; i < 10; i += 1) {
        if (i % 2 == 0) continue;
        print((string) i);
    }
    print("\n");
    string s = "outside";
    for (int i = 0; i < 5; i += 1) {
        string s = "outer";
        for (int j = 0; j < i; j += 1) {
            string s = "";
            print((string) i + " " + (string) j + "\n");
            {
                s = "inner";
                print(s + "\n");
            }
        }
        print(s + "\n");
    }
    print(s + "\n");
    return 0;
}