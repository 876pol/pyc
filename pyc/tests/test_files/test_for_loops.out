01234
024
13579
outer
1 0
inner
outer
2 0
inner
2 1
inner
outer
3 0
inner
3 1
inner
3 2
inner
outer
4 0
inner
4 1
inner
4 2
inner
4 3
inner
outer
outside
